src/textinput/example/index.ts

```diff
@@ -31,49 +31,49 @@ export class App extends AppBase<WidgetProperties> {
 				w(TextInput, {
 					key: 't1',
 					label: 'Name',
-					type: <TextInputType> 'text',
+					type: 'text',
```

Comment:
Do things like `ButtonType` and `TextInputType` still need to be exported now that we don't need to do this?